src/main/msp/msp.c

```diff
@@ -1076,6 +1080,38 @@ static bool mspCommonProcessOutCommand(int16_t cmdMSP, sbuf_t *dst, mspPostProce
     return true;
 }
 
+#ifdef USE_MSP_DISPLAYPORT_DISARM_DELAY
+
+#define DISARM_DELAY_MULTIPLIER_US 100000
+
+static void mspDisplayportDelayDisarm(mspDescriptor_t srcDesc, boxBitmask_t *flightModeFlags)
+{
+    static mspDescriptor_t displayPortMspDescriptor;
+    static bool displayPortMspArmState = false;
+    static bool descriptorInitialized = false;
+
+    if (!descriptorInitialized) {
+        descriptorInitialized = true;
+        displayPortMspDescriptor = getMspSerialPortDescriptor(displayPortMspGetSerial());
```

Comment:
It’s a compromise between speed and expected functionality here. Were somebody to switch which port was being used for the VTX then this could break until the next reset, but that could never affect operation in flight, or even an motor test, and so has no safety concerns.